src/components/InstallTabs/InstallTabs.scss

```diff
@@ -24,7 +24,7 @@
       width: 100%;
 
       &--selected {
-        background: var(--black9);
+        background: var(--black10);
```

Comment:
This reverts the changes done on the Install Node.js demo on the Homepage to how it was supposed to be (Figmas).